phpstan.neon

```diff
@@ -18,5 +18,7 @@ parameters:
 		- '#Call to method install\(\) on an unknown class PrestaShop\\PrestaShop\\Core\\Module\\ModuleManager.#'
 		- '#Call to method enable\(\) on an unknown class PrestaShop\\PrestaShop\\Core\\Addon\\Module\\ModuleManager.#'
 		- '#Call to method enable\(\) on an unknown class PrestaShop\\PrestaShop\\Core\\Module\\ModuleManager.#'
+		- '#Property Prestashop\\ModuleLibMboInstaller\\HttpClient::\$curl has no type specified.#'
+		- '#Method Prestashop\\ModuleLibMboInstaller\\Response::getBody\(\) return type has no value type specified in iterable type array.#'
```

Comment:
Is this ignored error still expected?

Reply:
For the cURL error, I can't add a type because it was only introduced in PHP 8.

Reply:
Yes, I was only referring to the second error.